scripts/qa_crowbarsetup.sh

```diff
@@ -4735,6 +4736,26 @@ function onadmin_addupdaterepo
     mkdir -p $UPR
 
     local repos=$UPDATEREPOS
+    # Extra repos can be added on demand to the ptf repo
+    # From the variable $UPDATEREPOS_EXTRA the first 'n' repos are added, where n is the parameter to this function
+    # So addupdaterepo can be called repeatedly. Contents of the extra repos are only added but never removed.
+    local extra_index
+    if [[ $extra_repos ]] ; then
+        case $extra_repos in
+            all|0)
+                extra_index='' ;;
+            [1-9]*([0-9]))
+                extra_index=$extra_repos ;;
+            *)
+                complain 13 "Invalid parameter for addupdaterepo. Valid are integers > 0 or the string 'all'. The value was: $extra_repos" ;;
+        esac
+        repos+="+$(echo $UPDATEREPOS_EXTRA | cut -d+ -f1-$extra_index)"
```

Comment:
like this: https://github.com/SUSE-Cloud/automation/commit/710e51149dadab86b67b761794c878d6032da2ce